### PR TITLE
Qaqc reporting

### LINKED ISSUE #107: QAQC reporting rollup in feature report and scenario report

### PR DESCRIPTION
**Pull request by vtnate**
### Pull Request Description

Reads workflow object for qaqc flags (if enabled by the user in the project workflow) and populates them in the feature reports.

### Checklist (Delete lines that don't apply)

- [ ] Unit tests have been added or updated
- [ ] Documentation has been modified appropriately
- [x] All ci tests pass (green)
- [x] An [ISSUE](https://github.com/urbanopt/urbanopt-reporting-gem/issues) has been created that this is addressing. Issues will get added to the Change Log when the change_log.rb script is run.
- [x] This branch is up-to-date with develop

### CODE REVIEW

**Review by rawadelkontar — APPROVED** [review]

File: lib/measures/default_feature_reports/measure.rb [hunk 1]
```diff
@@ -310,31 +310,40 @@ def feature_qaqc_flags(runner)
...
-              
-              if qaqc_flags[name]
-                qaqc_flags[name] += value
-              else 
-                qaqc_flags[name] = value
+              if qaqc_flags_list.include? name
+
+                # get value
+                # check if value, double, int, or bool
+                value_type = step_value.variantType.valueDescription
```
Comment (vtnate) [inline lib/measures/default_feature_reports/measure.rb:327]: Values of flag counts will only be integers, no?

File: lib/measures/default_feature_reports/measure.rb [hunk 2]
```diff
@@ -310,31 +310,40 @@ def feature_qaqc_flags(runner)
             result = measure_step.result.get
             puts " result = #{result}"
 
+            qaqc_flags_list = ['eui_reasonableness' , 'end_use_by_category',
```
Comment (vtnate) [inline lib/measures/default_feature_reports/measure.rb:313]: Add a comment explaining why we aren't checking if units == flag

**Review by vtnate — COMMENTED** [review]
Good update. Can you also remove any unnecessary schema changes I made earlier in this PR?

File: lib/measures/default_feature_reports/measure.rb [hunk 3]
```diff
@@ -41,12 +41,12 @@
 require 'urbanopt/reporting/default_reports'
 require 'csv'
 require 'benchmark'
+require 'json'
```
Comment (vtnate) [inline lib/measures/default_feature_reports/measure.rb:44]:
```suggestion
```

File: lib/urbanopt/reporting/default_reports/qaqc_flags.rb [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+# OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+# OF THE POSSIBILITY OF SUCH DAMAGE.
+# *********************************************************************************
+
+require 'json'
+require 'json-schema'
```
Comment (vtnate) [inline lib/urbanopt/reporting/default_reports/qaqc_flags.rb:42]: Do we need either of these requirements any more?

**Review by vtnate — COMMENTED** [review]
Couple imports look unused.

**Pull request merged**